### PR TITLE
User Manual Updates for 0.5

### PR DESCRIPTION
**Pull request by vansia43**
This PR addresses user manual updates for the various enhancements to the app. To note:

- The new Recurring Task page was reviewed and some minor updates made for style/consistency with the rest of the manual. It was also not displaying due to a missing tag and was not in the meson file, so I added that.
- The Plugins, Keyboard Shortcuts, and Preferences pages were all updated based on the additions of Hamster, Dark Mode, and new keyboard shortcuts.
- A few pages were updated based on the new task editor view (checkboxes and clickable subtasks, subheadings etc.). I also had to replace some of the screenshots using our screenshot dataset so that the recurring task icon and new task view were being displayed.

I believe this should cover all of the updates so far!

### CODE REVIEW

**Comment by vansia43** [comment]
Realized I put the arrow key references for e004b95. Tried to play around a bit to edit that commit and today is just not my day so I have another commit correcting that.... sorry for the messy history!

File: docs/user_manual/C/gtg-recurring-task.page [hunk 1]
```diff
@@ -87,15 +102,15 @@
...
-<p>The following shows examples of how recurring terms work. These examples are made using the <gui>Quick Add Entry</gui> but do not deffer in results from the Task Editor.</p>
+<p>Below are some examples of how recurring terms work. These examples are made using the <gui>Quick Add Entry</gui> but do not differ in result from the <gui>Task Editor</gui>.</p>
 
 <terms>
   <item>
      <title><em style="strong"><code>pay my taxes every:year</code></em></title>
-        <p>This task will repeat every year on the date which the task was created.</p></item>
+        <p>This task will repeat every year on the date that the task was created.</p></item>
   <item>
      <title><em style="strong"><code>Go out with my dog every:day start:next month</code></em></title>
```
Comment (diegogangl) [inline docs/user_manual/C/gtg-recurring-task.page:112]: `start:next month` isn't valid. Maybe `start:tomorrow`?

**Review by diegogangl — CHANGES_REQUESTED** [review]
Hey @vansia43, thanks for the work! I only found one small nit-pick.
Don't worry about the history :+1:

**Comment by vansia43** [comment]
This should be mostly it for the user manual. Note that I might end up making a few more minor updates based on when #525 is complete. There is an old sync services page that I marked as outdated last release since we didn't have any. I can reinstate that and check that the text is still relevant when that PR is finished. Also, I would make an update to Plugins when #521 is merged.

**Comment by diegogangl** [comment]
I could swear I had merged this. Sorry it took so long, thanks!

**Pull request merged**